C_CPP/Templates & Libraries/STL LISTS.md

```diff
@@ -0,0 +1,233 @@
+# STL LIST
+If we talk about the List in standard template library.
+<p> List is a sequence container that allows insertion and deletion operations in constant time from any place in the sequence and can iterate in both directions forward and backward but traversing is bit slower. </p>
+
```

Comment:
```suggestion

 - List is a sequence container that allows insertion and deletion operations in constant time from any place in the sequence and can iterate in both directions forward and backward but traversing is bit slower.
- It allows non-contiguous memory allocation.
```

Get rid of html tags in markdown !

Reply:
sir i use p tag for making my content more readable and presentable and sir now you can check my work i add all the changes that you proposed . please check my work @vasu-1

Reply:
Cool 🥇